lib/cocoapods-core/specification/consumer.rb

```diff
@@ -144,6 +144,10 @@ def user_target_xcconfig
       spec_attr_accessor :requires_app_host
       alias_method :requires_app_host?, :requires_app_host
 
+      # @return [String?] Name of the app host this spec requires
```

Comment:
remove `?`

Reply:
my original question about specifying this is an option still stands

Reply:
hm not sure what consumer does in the case its empty but I haven't seen this convention for documentation anywhere else.